releases/development/master/extra/barclamp_install.rb

```diff
@@ -109,15 +109,25 @@ def rm_tmpdir(tmpdir)
     puts "Barclamp at #{bc} has no name, skipping"
     next
   end
+  version = barclamp["barclamp"]["version"].to_i rescue 0
   order = 9999
```

Comment:
I think we don't need the rescue in both cases. IMHO ```order = barclamp["crowbar"]["order"] || 9999``` should be for both cases a good solution :)